model-ontology/src/ontology/Data/UpperModel.pins

```diff
@@ -286,6 +286,41 @@
 	(identifier "sequence_number")
 	(specMesg "The sequence number of the second axis of an Array_2D_Image must be set to 2."))
 
+([http%3A%2F%2Fpds.nasa.gov%2Finfomodel%2Fpds%23.pds%3ACitation_Information.100004635] of  Schematron_Rule
+
+	(alwaysInclude "false")
+	(attrNameSpaceNC "pds")
+	(attrTitle "author_list")
+	(classNameSpaceNC "pds")
+	(classSteward "pds")
+	(classTitle "Citation_Information")
+	(has_Schematron_Assert
+		[http%3A%2F%2Fpds.nasa.gov%2Finfomodel%2Fpds%23.pds%3ACitation_Information.100004635.101]
+		[http%3A%2F%2Fpds.nasa.gov%2Finfomodel%2Fpds%23.pds%3ACitation_Information.100004635.102])
+	(identifier "pds:Citation_Information")
+	(isMissionOnly "false")
+	(roleId "TBD_roleId")
+	(type "TBD_type")
+	(xpath "pds:Citation_Information"))
+
+([http%3A%2F%2Fpds.nasa.gov%2Finfomodel%2Fpds%23.pds%3ACitation_Information.100004635.101] of  Schematron_Assert
+
+	(assertMsg "The presence of \"pds:doi\" requires at least one of either \"List_Author\" or \"List_Editor\" classes.")

```

Comment:
@jshughes looks like the assert message was not updated like the schematron:

> "The presence of \"pds:doi\" requires at least one of either \"List_Author\" or \"List_Editor\" classes."

Should probably say > "The presence of \"pds:doi\" requires at least one of either \"author_list\", \"editor_list\", \"List_Author\" or \"List_Editor\" classes."

Reply:
``` 
      <sch:assert test=" if (pds:doi) then (pds:author_list or pds:editor_list or pds:List_Author or pds:List_Editor) else true()">
        The presence of "pds:doi" requires at least one of either "List_Author" or "List_Editor" classes.
      </sch:assert>
``` 

Do you agree that the above message was implemented exactly as ARaugh wrote and the DDWG approved.

Reply:
Will approve for now.